src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -130,8 +131,18 @@ public static Tag parseTag(String tag) throws ParseException {
     public static Set<Tag> parseTags(Collection<String> tags) throws ParseException {
         requireNonNull(tags);
         final Set<Tag> tagSet = new HashSet<>();
+        final Set<String> netWorthTags = Set.of("highnetworth", "midnetworth", "lownetworth");
+        boolean hasNetWorthTag = false;
```

Comment:
is this boolean field used anywhere else in the code?

Reply:
negative, it is meant for looping through to find existing networth tags